scripts/gunCard.js

```diff
@@ -0,0 +1,87 @@
+export class GunCardSheet extends ItemSheet {
+    
+    get template() {
+
```

Comment:
maybe something like

let sheet = gunCard.html
if (!game.user.isGM && this.actor.limited) {
    sheet = limited-sheet.html
}
return \`systems/vtt-bunkersandbadasses/template/${sheet}\`;

Reply:
Tthis was a template from a module and wont be necessary iin the future

Reply:
deleted, because its nor necessary for now